kickstart/PXELinux.erb

```diff
@@ -19,12 +19,10 @@ oses:
 DEFAULT linux
 
 LABEL linux
-    KERNEL <%= @kernel %>
-    <% if @host.operatingsystem.name == 'Fedora' and @host.operatingsystem.major.to_i > 16 -%>
-    APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac
-    <% elsif @host.operatingsystem.name != 'Fedora' and @host.operatingsystem.major.to_i >= 7 -%>
-    APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> network ks.sendmac
-    <% else -%>
-    APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> ksdevice=bootif network kssendmac
-    <% end -%>
-    IPAPPEND 2
+KERNEL <%= @kernel %>
+<% if (@host.operatingsystem.name == 'Fedora' and @host.operatingsystem.major.to_i > 16) or (@host.operatingsystem.name != 'Fedora' and @host.operatingsystem.major.to_i >= 7) -%>
+APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> network inst.ks.sendmac
```

Comment:
Yeah both fixed. I don't see `network` in the git repo and docs, so it is likely gone. The RHEL7 docs do not say a word tho.